### DISCUSSION TITLE
Is the current firmware 3.3.0b5 supposed to be compatible with code.pybricks.com

### DISCUSSION THREAD

**Discussion by BertLindeman** [pybricks/support#1082] [Q&A] [answered]
**short**
Running Spike Prime with firmware 3.3.0b5 on code.pybricks.com.
No print data seen.

**elaborated**
Was testing for #1081 on a Spike prime and an Inventor hub.
Happened to have a browser with both beta.pybricks.com and code.pybricks.com next to each other. 
So tried the "other" hub simultaneous on the "other" pybricks tab.

Running Google Chrome Version 113.0.5672.127 (Official Build) (64-bit) on windows 11.

So tried to run the beta firmware on code.pybricks.com.
 
But print output does not reach the pybricks output window.
The program seems to run fine though.
To prove that the program runs I display a counter on the matrix.

```python

from pybricks.hubs import PrimeHub
from pybricks import version
hub = PrimeHub()
print(version)

i = -99
while True:
    i += 1
    print("Hello World Spike", i)
    if i > 99:
        i=-99
    hub.display.number(i)

```

**Comment by BertLindeman** [comment]
Not sure yet, but it might be the output goes to the "other" tab

**Comment by laurensvalk** [comment, marked as answer]
> Is the current firmware 3.3.0b5 supposed to be compatible with code.pybricks.com

I'm afraid not. Programs will run, but you do not see any output in the output window.

So we have two options:
1. Make this clearer, release a patched Pybricks Code with a warning, etc.
2. Release 3.3 asap. :rocket: 

We're currently going for option 2.

**Reply by BertLindeman** [reply]
> > Is the current firmware 3.3.0b5 supposed to be compatible with code.pybricks.com
> 
> I'm afraid not. Programs will run, but you do not see any output in the output window.
> 
> So we have two options:
> 
>     1. Make this clearer, release a patched Pybricks Code with a warning, etc.
> 
>     2. Release 3.3 asap. 🚀
> 
> 
> We're currently going for option 2.

YES!

**Comment by BertLindeman** [comment]
clear answer,so close
